client/src/components/Main/InviteeListItem/index.css

```diff
@@ -0,0 +1,6 @@
+.react-icons {
```

Comment:
I haven't personally dealt with react-icons yet, but could you do some of this in a styled-component? It looks like you are already passing the color to the IconContext.Provider, so maybe you could wrap the img in a component that has the height, position, and right properties? 

It would just be nice to have our project consistent throughout and not have some styling done in css files and the rest in styled-components.